Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV EXTRA_PACKAGES="relstorage==3.4.5 psycopg2==2.9.3 python-ldap==3.4.0"
 
 RUN <<EOT
     apt-get update
+    apt-get -y upgrade
```

Comment:
I don't think we need it in the intermediate image.

Reply:
@jensens If there is a security fix for a build tool, I would like it to be applied before I run the build tool and the exploit steals my build secrets.